ci/Jenkinsfile-kubernetes

```diff
@@ -23,15 +23,19 @@ def readPropertiesFile(file) {
     return properties
 }
 
-def runCommandWithK8sDebugInfo(GString script) {
+def runCommandWithK8sDebugInfo(GString script, boolean suppressFailure = true) {
```

Comment:
`false` would seem like a more reasonable default

Reply:
ok, will change it, thanks!